lockfile_generator/src/yarn.rs

```diff
@@ -11,7 +12,7 @@ impl Generator for Yarn {
         "yarn.lock"
     }
 
-    fn command(&self) -> Command {
+    fn command(&self, _manifest_path: &Path) -> Command {
         let mut command = Command::new("yarn");
         command.args(["install", "--mode=update-lockfile"]);
```

Comment:
I forgot to add that but I believe our extensions do it already. This should not be included in this PR though.

Reply:
This is the only instance of `skip-build` in the repo currently: https://github.com/phylum-dev/cli/blob/main/extensions/yarn/main.ts#L165-L184
It _is_ in the `yarn` extension, but does not appear to be related to generating a lockfile from a manifest. The [part of the extension that does that](https://github.com/phylum-dev/cli/blob/main/extensions/yarn/main.ts#L238-L249) has the `update-lockfile` mode but not the `skip-build` mode. Maybe that is desired when running in a sandbox so that the bad scripts will expose themselves.

Reply:
https://github.com/phylum-dev/cli/pull/1079